### ISSUE TITLE
Create a fallback strategy for missing emoji

### ISSUE THREAD

**Issue by sskhandek** [sskhandek/react-native-emoji-input#5]
Some versions of iOS and Android do not support all the emoji unicode characters. Having a fallback emoji character upon detection of the missing emoji would replace it with a popular open source emoji font like Twemoji or EmojiOne

**Comment by aaeronn** [comment]
Yes, it should be on priority. Missing emoji's makes the entire emoji block look ugly

**Comment by sskhandek** [comment]
Thanks @aaeronn for the feedback. We're open to PRs, but we're trying our best to come up with a solution in the next week or so. We have been busy with conference travel 😀

**Comment by darekg11** [comment]
I have created a fork with a simple solution to that.
Since emoji support varies across devices and manufacturer I have added a props that filters Emojis by Unicode version release. So for example you can display only the most basic ones which were included in version `6.0`. I know this si not perfect but at least on most Android devices I have all emojis without any missing ones.
Would you be interested in PR containing that?

**Comment by sskhandek** [comment]
cc: @rijn 
That'd be great. Thank you very much @darekg11

**Comment by darekg11** [comment]
@sskhandek I will create a PR on the weekend.
